Sources/TuistLoader/Models+ManifestMappers/Target+ManifestMapper.swift

```diff
@@ -93,6 +93,8 @@ extension TuistGraph.Target {
         let additionalFiles = try manifest.additionalFiles
             .flatMap { try TuistGraph.FileElement.from(manifest: $0, generatorPaths: generatorPaths) }
 
+        let isExternal = externalDependencies.keys.contains(name)
```

Comment:
If I am not wrong we are serializing the `TuistGraph.Project` in `graph.json`, if that's the case, we can set this flag during `tuist dependencies fetch` instead of having to map it here

Reply:
Actually, maybe we can pass the boolean as input of this method, as we call it from `ManifestModelConverter:96` where we map external projects, we can pass `true` in that case and false in the rest. WDYT?